.github/workflows/long-running-azure.yaml

```diff
@@ -368,6 +379,12 @@ jobs:
           export PATH=$GITHUB_WORKSPACE/bin:$PATH
           which rad || { echo "cannot find rad"; exit 1; }
 
+          echo "*** Uninstalling existing Radius installation ***"
+          rad uninstall kubernetes
+
+          echo "*** Deleting radius-system namespace ***"
+          kubectl delete namespace radius-system --ignore-not-found
```

Comment:
I do not think this is right approach. it is still workaround. upgrade scenario should work here.